.github/workflows/test-helm-chart.yaml

```diff
@@ -69,7 +66,7 @@ jobs:
       - name: "(Upgrade) Install helm diff"
         if: matrix.test == 'upgrade'
         run: |
-          helm plugin install https://github.com/databus23/helm-diff
+          helm plugin install https://github.com/databus23/helm-diff --version 3.1.3
```

Comment:
can we do sth like `--version stable` do they have a tag?

Reply:
I don't think they do but I'm ok to go back on `master`/`main` once they fix